src/runner/browser-env/vite/browser-modules/constants.ts

```diff
@@ -22,3 +22,5 @@ export const MAX_ARGS_LENGTH = 50;
 // used from - https://github.com/jestjs/jest/blob/726ca20752e38c18e20aa21740cec7aba7891946/packages/pretty-format/src/plugins/AsymmetricMatcher.ts#L11-L14
 export const ASYMMETRIC_MATCHER =
     typeof Symbol === "function" && Symbol.for ? Symbol.for("jest.asymmetricMatcher") : 0x13_57_a5;
+
+export const CONSOLE_METHODS = ["log", "info", "warn", "error", "debug"] as const;
```

Comment:
What about `trace`?

`table` also could be usefull.

Reply:
> What about trace? Would it work?

It works, but stack trace is not quite helpful, because it called inside worker.

> table also could be usefull.

Ok, I add it.